javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md

```diff
@@ -119,10 +119,10 @@ console.log(proxy.enki)
 ```
 
 * Proxy
-* revokable
+* revocable
 * get
 * proxy
-* revokable
+* rev
```

Comment:
revokable should work too

Reply:
As a variable name, it's too similar to `revocable`, makes the code confusing, no?

As a term it's technically correct but the api is called `revocable` (and that spelling is more popular) so I figured it's less confusing to just use that throughout